otcextensions/sdk/auto_scaling/v1/instance.py

```diff
@@ -59,6 +61,45 @@ class Instance(_base.Resource):
     #: AutoScaling instance create time
     create_time = resource.Body('create_time')
 
+    @classmethod
+    def find(cls, session, name_or_id, ignore_missing=True, **params):
```

Comment:
Hi Irina, 
please describe briefly in one or more additional comments in the find method of this resource to understand quickly why a change was necessary. In case of an upcoming change a following dev knows why and what has been done.

Reply:
Done